src/test/java/edu/kis/powp/jobs2d/events/CopyCommandListener.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.DefaultCompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CopyCommandListener implements ActionListener {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        List<DriverCommand> commands = new ArrayList<DriverCommand>();
+        commands.add(new SetPositionCommand(-20, -50));
+        commands.add(new OperateToCommand(-20, -50));
+        commands.add(new SetPositionCommand(-20, -40));
+        commands.add(new OperateToCommand(-20, 50));
+        commands.add(new SetPositionCommand(0, -50));
+        commands.add(new OperateToCommand(0, -50));
+        commands.add(new SetPositionCommand(0, -40));
+        commands.add(new OperateToCommand(0, 50));
+        commands.add(new SetPositionCommand(70, -50));
+        commands.add(new OperateToCommand(20, -50));
+        commands.add(new OperateToCommand(20, 0));
+        commands.add(new OperateToCommand(70, 0));
+        commands.add(new OperateToCommand(70, 50));
+        commands.add(new OperateToCommand(20, 50));
+        DriverCommand driverCommand = CommandsFeature.deepCopyDriverCommand(new DefaultCompoundCommand(commands));
```

Comment:
If you want to test DEEP cloning you must put a compound command inside compound command. Next you can try to modify the original to test if copy changes.